tests/neg-macros/i7048e.scala

```diff
@@ -16,18 +16,18 @@ abstract class Test {
       import t.given
       println(summon[Type[t.T]].show)
       // val r = '{Option.empty[t.T]} // access to value t from wrong staging level
-      val r2 = '{Option.empty[${t.T}]} // works
+      val r2 = '{Option.empty[t.T.Underlying]} // works
     }
 
     {
-      val r1 = '{Option.empty[${T}]} // works
-      val r2 = '{Option.empty[List[${T}]]} // works
-      val r3 = '{summon[Type[${T}]]} // error: is not stable
-      val r4 = '{summon[${T} <:< Any]} // error: is not stable
+      val r1 = '{Option.empty[T.Underlying]} // works
+      val r2 = '{Option.empty[List[T.Underlying]]} // works
+      val r3 = '{summon[Type[T.Underlying]]} // error: is not stable
+      val r4 = '{summon[T.Underlying <:< Any]} // error: is not stable
```

Comment:
It's a little surprising that `List[T.Underlying]` works but `summon[Type[T.Underlying]]]` is an error.

Reply:
This one was subtle.
```
[error] (Test.this.given_Type_T : => quoted.Type[Test.this.T]) is not a valid type witness, since it is not an immutable path
```

The issue is that `given Type[T] = getT` is not stable. This is where I usually write `given getT.type = getT` to not lose the path in the given.

The other cases happen to work because we only referred to `T.Underlying` and never needed to head some type. This is the expected behavior.